spec/jstests-spec.js

```diff
@@ -195,3 +195,75 @@ describe("Initializing tooltips", function() {
   });
 });
 
+describe("Pressing next and previous buttons associated with a graph", function() {
+  it("generates correct fetch request when next/previous buttons are pressed", function() {
+
+    const prevButton = document.createElement("button");
+    prevButton.id = "prevbutton";
+    prevButton.onclick = () => navigateGraph(-1);
+    const nextButton = document.createElement("button");
+    nextButton.id = "nextbutton";
+    nextButton.onclick = () => navigateGraph(1);
+    document.body.appendChild(prevButton);
+    document.body.appendChild(nextButton);
+
+    expect(currGraphNum).toBe(0);
+    expect(numMutations).toBe(3);
```

Comment:
This currGraphNum and numMutations are both globally exported variables. If you end up changing the value of numMutations, you'd have to make sure to change this test as well. 
Instead, why not export `initializeNumMutations(num)` that sets this variable? You then have the flexibility to set it to anything in tests, and can also call this method when you actually pass in the total number of mutations to script.js? 
Same goes for initializing currGraphNum.

Reply:
Ohh that's smart, and I think I understand what you mean! Made the changes, but let me know if it's not what you were expecting!